include/OpenSpaceToolkit/Astrodynamics/Flight/System/Dynamics/Thruster/ConstantThrustThruster.hpp

```diff
@@ -0,0 +1,167 @@
+/// Apache License 2.0
+
+#ifndef __OpenSpaceToolkit_Astrodynamics_Flight_System_Dynamics_Thruster_ConstantThrustThruster__
+#define __OpenSpaceToolkit_Astrodynamics_Flight_System_Dynamics_Thruster_ConstantThrustThruster__
+
+#include <OpenSpaceToolkit/Core/Types/Real.hpp>
+
+#include <OpenSpaceToolkit/Mathematics/Objects/Vector.hpp>
+
+#include <OpenSpaceToolkit/Physics/Data/Direction.hpp>
+#include <OpenSpaceToolkit/Physics/Time/Instant.hpp>
+#include <OpenSpaceToolkit/Physics/Units/Mass.hpp>
+
+#include <OpenSpaceToolkit/Astrodynamics/Flight/System/Dynamics/Thruster.hpp>
+#include <OpenSpaceToolkit/Astrodynamics/Flight/System/SatelliteSystem.hpp>
+#include <OpenSpaceToolkit/Astrodynamics/Trajectory/LocalOrbitalFrameDirection.hpp>
+#include <OpenSpaceToolkit/Astrodynamics/Trajectory/State.hpp>
+
+namespace ostk
+{
+namespace astro
+{
+namespace flight
+{
+namespace system
+{
+namespace dynamics
+{
+namespace thruster
+{
+
+using ostk::core::types::Real;
+
+using ostk::math::obj::Vector3d;
+using ostk::math::geom::d3::trf::rot::RotationMatrix;
+
+using ostk::physics::time::Instant;
+using ostk::physics::units::Mass;
+
+using ostk::astro::flight::system::SatelliteSystem;
+using ostk::astro::flight::system::PropulsionSystem;
+using ostk::astro::flight::system::Dynamics;
+using ostk::astro::trajectory::State;
+using ostk::astro::trajectory::LocalOrbitalFrameDirection;
+
+/// @brief                      Define the acceleration experienced by a point mass due to a constant thrust thruster
+/// dynamics
+
+class ConstantThrustThruster : public Thruster
```

Comment:
```suggestion
class ConstantThrust : public Thruster
```
Nit: Maybe rename to ConstantThrust?

Reply:
@phc1990 @kyle-cochran Thoughts?

Reply:
Yeah okay good point. My view was that `ConstantThrust` alone (and same for dynamics children) does not necessarily convey the meaning of `Thruster` (or `Dynamics` for the others). But that can be made clear in the tutorials I think